File: CodeSignal/arrayMaximalAdjacentDifference.java
int solution(int[] inputArray) {
    
    int maxDiff = 0;
    
    for (int i = 0; i < inputArray.length - 1; i++) {
        maxDiff = Math.max(Math.abs(inputArray[i] - inputArray[i+1]), maxDiff);
    }
    
    return maxDiff;
}

File: CodeSignal/arrayMaximalAdjacentDifference.py
def solution(inputArray):
    res = 0
    left = 0
    while left < len(inputArray) - 1:
        right = left + 1
        if abs(inputArray[right] - inputArray[left]) > res:
            res = abs(inputArray[right] - inputArray[left])
            print("res", res)
        left += 1
        
    return res